cli/src/cmd_auth.rs

```diff
@@ -699,107 +594,107 @@ mod tests {
+        use assert_cmd::Command;
+        use httpmock::{Method::GET, MockServer};
+        use predicates::str;
+
+        let server = MockServer::start();
+        let oxide_mock = server.mock(|when, then| {
+            when.method(GET)
+                .path("/v1/me")
+                .header("authorization", "Bearer oxide-token-1111");
+            then.status(200).json_body_obj(&oxide::types::CurrentUser {
+                display_name: "privileged".to_string(),
+                id: "001de000-05e4-4000-8000-000000004007".parse().unwrap(),
+                silo_id: "d1bb398f-872c-438c-a4c6-2211e2042526".parse().unwrap(),
+                silo_name: "funky-town".parse().unwrap(),
+            });
         });
-    });
-
-    // Validate authenticated credentials.
-    Command::cargo_bin("oxide")
-        .unwrap()
-        .arg("auth")
-        .arg("status")
-        .env("OXIDE_HOST", server.url(""))
-        .env("OXIDE_TOKEN", "oxide-token-1111")
-        .assert()
-        .success()
-        .stdout(str::contains(format!(
-            "{}: [{}, {}]",
-            server.url(""),
-            format_args!(
-                "\"Logged in to {} as 001de000-05e4-4000-8000-000000004007\"",
-                server.url("")
-            ),
-            "\"Token: *******************\"",
-        )));
-
-    // Validate authenticated credentials with the token displayed in plain text.
-    Command::cargo_bin("oxide")
-        .unwrap()
-        .arg("auth")
-        .arg("status")
-        .env("OXIDE_HOST", server.url(""))
-        .env("OXIDE_TOKEN", "oxide-token-1111")
-        .arg("-t")
-        .assert()
-        .success()
-        .stdout(str::contains(format!(
-            "{}: [{}, {}]",
-            server.url(""),
-            format_args!(
-                "\"Logged in to {} as 001de000-05e4-4000-8000-000000004007\"",
-                server.url("")
-            ),
-            "\"Token: oxide-token-1111\"",
-        )));
-
-    // Assert that both commands hit the mock.
-    oxide_mock.assert_hits(2);
-
-    let oxide_mock = server.mock(|when, then| {
-        when.header("authorization", "Bearer oxide-token-1112");
-        then.status(401).json_body_obj(&oxide::types::Error {
-            error_code: None,
-            message: "oops".to_string(),
-            request_id: "42".to_string(),
+
+        // Validate authenticated credentials.
+        Command::cargo_bin("oxide")
+            .unwrap()
+            .arg("auth")
+            .arg("status")
+            .env("OXIDE_HOST", server.url(""))
+            .env("OXIDE_TOKEN", "oxide-token-1111")
+            .assert()
+            .success()
+            .stdout(str::contains(format!(
+                "{}: [{}, {}]",
+                server.url(""),
+                format_args!(
+                    "\"Logged in to {} as 001de000-05e4-4000-8000-000000004007\"",
+                    server.url("")
+                ),
+                "\"Token: *******************\"",
```

Comment:
going to need an update

Reply:
Removed with https://github.com/oxidecomputer/oxide.rs/pull/727